File: go/src/leetcode/editor/cn/0169_majority_element .go
package main

import "sort"

func main() {
	println("169、majority_element")
}

// leetcode submit region begin(Prohibit modification and deletion)
func majorityElement(nums []int) int {
	sort.Ints(nums)
	return nums[len(nums)/2]
}

//leetcode submit region end(Prohibit modification and deletion)

File: rust/src/leetcode/editor/cn/_0169_majority_element.rs
#[cfg(test)]
mod tests {
    #[allow(unused_imports)]
    use super::*;

    #[test]
    fn test() {
        println!("169、majority_element");
    }
}

#[allow(dead_code)]
struct Solution;

//leetcode submit region begin(Prohibit modification and deletion)
impl Solution {
    #[allow(dead_code)]
    pub fn majority_element(mut nums: Vec<i32>) -> i32 {
        nums.sort_unstable();
        nums[nums.len() / 2]
    }
}
//leetcode submit region end(Prohibit modification and deletion)
